packages/slice-machine/server/src/api/pushChanges/index.ts

```diff
@@ -37,6 +40,33 @@ export default async function handler({
     const { localSlices, remoteSlices, localCustomTypes, remoteCustomTypes } =
       await fetchModels(client, cwd, manifest.libraries);
 
+    const customTypesWithInvalidSliceReferences = localCustomTypes.filter(
```

Comment:
could you extract it in a function please 🙏 to make it clear on what's happening and the goal of the code